websites/S/Sitting on Clouds Radio/dist/metadata.json

```diff
@@ -12,11 +12,12 @@
 	],
 	"service": "Sitting on Clouds Radio",
 	"description": {
-		"en": "Video game music radio from the Sitting on Clouds Soundtrack Collection",
-		"nl": "Video game muziek radio van de Sitting on Clouds Soundtrack Collectie"
+		"en": "Video game music radio from the Sitting on Clouds Soundtrack Collection.",
+		"fr": "Radio de musiques de jeux vidéos de la collection de bandes son de Sitting on Clouds.",
+		"nl": "Video game muziek radio van de Sitting on Clouds Soundtrack Collectie."
 	},
 	"url": "radio.sittingonclouds.net",
-	"version": "1.2.0",
+	"version": "1.3.0",
```

Comment:
```suggestion
	"version": "1.2.1",
```

Reply:
Resolved in 24ca84dc922fbfccb07bcb5643f0f12a8392f9d8